ProteoformSuiteInternal/Lollipop.cs

```diff
@@ -513,10 +513,11 @@ public static void make_ee_relationships()
         //METHOD FILE
         public static string method_toString()
         {
-            return String.Join(System.Environment.NewLine, new string[] {
-                "LoadDeconvolutionResults|deconvolution_file_names\t" + String.Join("; ", Lollipop.identification_files().Select(s => s.filename).ToArray<string>()),
-                "LoadDeconvolutionResults|neucode_labeled\t" + neucode_labeled.ToString(),
-                "CorrectionFactors|correction_file_names\t" + String.Join("; ", Lollipop.calibration_files().Select(s => s.filename).ToArray<string>()),
+            string method = String.Join(System.Environment.NewLine, new string[] {
```

Comment:
Actually come to think of it -- for opening it would be nice if those files then showed up in the DGV for load decon results and you could look through them. This area still needs work to have all those different types of files listed/show up so I'll fix that soon.